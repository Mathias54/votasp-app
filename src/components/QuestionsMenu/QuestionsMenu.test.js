import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import QuestionMenu from './QuestionsMenu';

describe('QuestionsMenu', () => {
  it('should match a snapshot', () => {
    const questionMenu = shallow((
      <QuestionMenu />
    ))

    expect(toJson(questionMenu)).toMatchSnapshot();
  })

  it('should toggle questions board when click on questions button', () => {
    const questionMenu = shallow((
      <QuestionMenu />
    ))

    questionMenu.find('.questions-button').simulate('click');
    expect(questionMenu.find('.questions-board')).toHaveLength(1)

    questionMenu.find('.questions-button').simulate('click');
    expect(questionMenu.find('.questions-board')).toHaveLength(0)

  })
})
