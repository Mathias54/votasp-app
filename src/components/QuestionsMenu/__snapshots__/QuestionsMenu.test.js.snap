// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionsMenu should match a snapshot 1`] = `
<div
  className="questions-menu"
>
  <div
    className="questions-header"
  >
    <button
      className="questions-button"
      onClick={[Function]}
      type="button"
    >
      Quest√µes
    </button>
  </div>
</div>
`;
